solidity/contracts/lib/verifier_check_nullifiers.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
can the script be enhanced to skip certain circuit for the solidityverifier export step? the `check_nullifiers` circuit is only useful in off-chain use cases like a centralized component such as a notary to be able to validate nullifiers, in order to support anonymity in notarized transactions

Reply:
I think maybe this one can be a follow-on PR, we need to consider whether the script should be smart enough to be aware of "profiles". I don't want a bash script to get too complex. Maybe we should switch to a js script at that stage.

Reply:
coverted the script to use JS